e2e/disruptors/pod_e2e_test.go

```diff
@@ -39,6 +39,14 @@ func Test_PodDisruptor(t *testing.T) {
 		_ = cluster.Cleanup()
 	})
 
+	err = cluster.Load(
```

Comment:
I find this code hard to read, I think we could offer a helper function for extracting the images, but this can be in a follow-up PR.

Reply:
Agreed, it is a bit confusing. I also considered storing those images on an exported constant, and have both fixtures and this code use those constants, but seemed easier to forget than this.